truss/types.py

```diff
@@ -71,3 +72,25 @@ def from_dict(patch_details: Dict) -> "PatchDetails":
                 Patch.from_dict(patch_op) for patch_op in patch_details["patch_ops"]
             ],
         )
+
+
+class PatchRequest(BaseModel):
+    """Request to patch running container"""
+
+    hash: str
+    prev_hash: str
+    patches: List[Patch]
+
+    def to_dict(self):
```

Comment:
Is this different from the dict method that comes with pydantic. Same for from_dict, pydantic has parse_obj.

Reply:
To use `dict` or `parse_obj` form `pydantic` we would need to convert all of the `Patch` helper types upstream to be typed with `pydantic`. Let me take this on in a follow up